GiniSwitchSDK/GiniSwitchSDK/Classes/ExtractionValue.swift

```diff
@@ -6,19 +6,19 @@
 //
 //
 
-struct ExtractionValue {
+public struct ExtractionValue {
     
-    let value:AnyObject
-    let unit:String?
+    public let value:AnyObject
+    public let unit:String?
     
     static let valueKey = "value"
     static let unitKey = "unit"
     
-    var valueString:String {
+    public var valueString:String {
```

Comment:
Here, the member is actually a computed property so by definition it has to be variable. The compiler cannot guarantee it's not going to change, I guess.